packages/message-manager/CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Added
+- Add `getAllMessages` and `setMetadata` methods to message managers ([#1364](https://github.com/MetaMask/core/pull/1364))
+  - A new optional `metadata` property has been added to the message type as well
+- Add support for deferred signing ([#1364](https://github.com/MetaMask/core/pull/1364))
+  - If the parameter `deferSetAsSigned` is set, the message won't be set as signed when the keyring is asked to sign it
```

Comment:
It seems like this particular point belongs in `@metamask/signature-controller` since that's where `deferSetAsSigned` is used. Maybe we should leave this point out and just mention that messages have an optional `deferSetAsSigned` property or something?

Reply:
Addressed here: 4b30357adee6c2ec29d8c411bffa02f2c653244c